### PR TITLE
Add ListClients and DeleteClient operations

### LINKED ISSUE #93: Add a new ResponseStatus for admin operations
If a normal user tries to perform an admin operation, a specific response status should be returned. I can propose:

* name: `AdminOperation`
* value: 21
* description: The operation requires admin privilege

### LINKED ISSUE #94: Add ListClients and DeleteClient structures
After updating the submodule to include [this](https://github.com/parallaxsecond/parsec-operations/issues/26).

### PR DESCRIPTION
**Pull request by hug-dev**
Also adds the new response status code
Fix #93 
Fix #94

### CODE REVIEW

**Review by ionut-arm — COMMENTED** [review]
Changes look good, only one question (maybe I didn't see it?): don't you need another method on `Opcode` - `is_admin`?

**Comment by hug-dev** [comment]
> Changes look good, only one question (maybe I didn't see it?): don't you need another method on Opcode - is_admin?

Oh that's a good idea. I was thinking of adding the check into every case that needs it in the back end handler but that seems more optimal to do it once for all, will add.

File: src/requests/mod.rs [hunk 1]
```diff
@@ -129,6 +133,37 @@ impl Opcode {
...
+            Opcode::Ping => false,
+            Opcode::ListProviders => false,
+            Opcode::ListOpcodes => false,
+            Opcode::ListAuthenticators => false,
+            Opcode::ListKeys => false,
+            Opcode::ListClients => true,
+            Opcode::DeleteClient => true,
             Opcode::PsaGenerateKey => false,
             Opcode::PsaDestroyKey => false,
             Opcode::PsaSignHash => false,
```
Comment (ionut-arm) [inline src/requests/mod.rs:169]: Sorry for being nitpicky - could you group them together with `|`? It's easier to check the subset that returns `true` if they're organised like that.

**Review by ionut-arm — APPROVED** [review]

File: src/requests/mod.rs (hunk 1 above)
Comment (hug-dev) [reply src/requests/mod.rs]: Absofruitly 🍌 Just done it

**Pull request merged**